### DISCUSSION TITLE
Google 小姐說故事給你聽~

### DISCUSSION THREAD

**Discussion by henrywen547** [1101-GS4719-C/Python-Community-GS4719-C#565] [General]
不知道大家有沒有以下困擾(我自己是有啦)
看文章看的好想睡覺，卻只能一直撐著最後就度估了，最後就算內容有看完，也不一定記得文章在說什麼...
後來，我就找到了Chrome的擴充功能-Read Aloud: 文字語音朗讀助理，它來幫我念就輕鬆多了!
![Read Aloud](https://lh3.googleusercontent.com/_l4UaD5Gwub2AwFZdomRpfMD2737y0Ow6k2sfVuWdm0fQ0iGSdEfhd9X77taeQ-0VM6Wi3HnpecLWUAI9uHq11TSktw=w128-h128-e365-rj-sc0x00ffffff)
## 功能介紹
_______
### **播放/暫停**
反白之後按alt+P
### **停止**
alt+O
### **快轉/倒帶**
alt+句號/alt+逗號
______
**(快捷鍵和語速都可以自己設定~)**

**Comment by cordydeng** [comment]
喔喔喔感覺是很棒的功能ㄟ , 不知道說出來的聲音有沒有抑揚頓挫 , 還是只是死板板的語音?

**Comment by Sazmafuji** [comment]
這擴充功能好棒，不用再用看的去看預報了

**Comment by HsuChen88** [comment]
最近覺得眼睛盯著螢幕的時間偏長，有這個的話要抄東西的時候直接聽寫感覺好多了，以免用眼過度。

**Comment by hapiwangy** [comment]
真是個方便的功能!!以後可以同時做更多事情了

**Comment by ThomasMeng0515** [comment]
感覺很方便，有點像聽podcast說書的感覺

**Comment by PoooYen** [comment]
我一直覺得google小姐的抑揚頓挫怪怪的哈哈哈

**Comment by bloodtrail0511** [comment]
我前陣子剛好也裝了這個擴充，使用起來真的方便

**Comment by HuanYuuuuuu** [comment]
欸這個感覺很不錯

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，使用起來感覺很好。

**Comment by yi-1111** [comment]
謝謝分享，感覺又多了一個可以一心多用的工具

**Comment by yucheng1028** [comment]
可惜google 小姐得聲音還是沒有siri的香

**Comment by d26488399** [comment]
感謝分享，真的好好用

**Comment by Yuhsin000** [comment]
GOOGLE小姐講故事我會不小心聽得太專心(抓他奇怪的語調)，反而睡不著

**Comment by Lai27330** [comment]
感覺好方便，這樣就能一邊聽文章，一邊去做其他事順便休息眼睛了，但如果死板板的音調會有點想睡，謝謝分享。

**Comment by lx3211** [comment]
我高中同學有人學google小姐超像，我要跟她分享這個！

**Comment by sys511613** [comment]
現在edge瀏覽器都有內建這個功能了，把文字選起來按右鍵就可以看到"大聲朗讀選取項目"

**Comment by minniewei** [comment]
感謝分享啦啦啦，感覺真的超級方便

**Comment by treeann** [comment]
google小姐說話的聲音很好笑哈哈

**Comment by stan890425** [comment]
這樣讀paper就有人唸給你聽了~~

**Comment by jcpan821** [comment]
感謝分享~~

**Comment by alexthebigpp** [comment]
google小姐asmr，感覺好棒。

**Comment by 110409538** [comment]
感謝分享!有時候閱讀的文章比較長,要一直看著手機螢幕密密麻麻的文字,對眼睛也是一種負擔.

**Comment by Ben6uo** [comment]
這種東西太合我這種懶人了

**Comment by etin2468** [comment]
可是改成用聽的，我一樣會睡著ㄟ

**Comment by Lynn12378** [comment]
很方便，但是Google小姐的聲音真的會讓人很容易分心XDD

**Comment by Jimmy911218** [comment]
很有趣但小姐的聲音xd

**Comment by Zoe06060606** [comment]
這樣感覺好像在聽podcast 🤣

**Comment by harrylee1971** [comment]
google還有很多好用的擴充軟體欸，他們讓我們的生活變得比較順利

**Comment by BlahFish1006** [comment]
哇......平常用看的讀論文都得的很痛苦很想睡覺逃避，
不知道用這個擴充功能讀論文會不會效果比較好，還能順便練習英文聽力，
或是成為一個特別棒的睡前有聲故事~ 有效幫助失眠的人快速進入學術夢鄉。
